en-US/O.xml

```diff
@@ -108,7 +108,29 @@
 			</listitem>
 
 		</varlistentry>
-<!-- Add entries for "on-prem", "on-premise", "on-premises"? -->
+
+<!-- Advice differs from IBM Style, which says to use "on premises", "on-premises". -->
+		 <varlistentry id="on premises">
+			<term>on premise, on-premise</term>
+			 <listitem>
+				<para>
+					<emphasis>adjective, adverb</emphasis> Write as shown. Use "on-premise" as an adjective before a noun. Otherwise, use "on premise." Do not use "on premises", "on-premises."
```

Comment:
We don't have our design and layout standards nailed down yet, but we are separating the word forms (noun, adjective, etc.), into separate lines, such as:
_adj._ Use "on-premise" as an adjective before a noun.
_adv._ Use "on premise" as an adverb.
When I redo the book I autogenerate the id's, too, but it might be a good idea to avoid spaces.
I also tend to cross-reference between similar and related entries. Just something simple: See also `<xref linkend="id-value">`